examples/maps_heatmaps/src/index.ts

```diff
@@ -69,12 +69,10 @@ function initMap(): void {
     //TODO: check what should be the default radius value.
     let newRadius = 2;
     if (bounds) {
-      newRadius = utils.getRadius(
-        bounds.getCenter().lat(),
-        bounds.getCenter().lng(),
-        bounds.getNorthEast().lat(),
-        bounds.getNorthEast().lng()
-      );
+      newRadius = google.maps.geometry.spherical.computeDistanceBetween(
+        bounds.getCenter(),
+        bounds.getNorthEast()
+      ) * 0.000621371192; //convert to miles
```

Comment:
Separate the radius computation and the conversion to miles to two lines

Reply:
seperated.